packages/playground/src/components/main_footer.vue

```diff
@@ -82,7 +82,18 @@
         </v-col>
       </v-row>
       <v-divider class="my-3 w-25 mx-auto" />
-      <div class="text-center mb-12">{{ new Date().getFullYear() }} — ThreeFoldTech</div>
+
+      <div class="mb-12 d-flex justify-center text-subtitle-2">
+        <p>
+          &#169; {{ new Date().getFullYear() }} ThreeFoldTech
+          <span v-if="version !== 'No version to show'">
```

Comment:
might be a good idea to place 'No version to show' in a constant @AhmedHanafy725

Reply:
the idea is to show the version to the user, if there is no version, there is no need to tell him that 
@AhmedHanafy725 what do you think?

Reply:
> might be a good idea to place 'No version to show' in a constant @AhmedHanafy725

Done @amiraabouhadid can you check please